scripts/read_ledger

```diff
@@ -153,15 +156,26 @@ def make_copy_of_ledger(data_dir):
 
 if __name__ == '__main__':
     args = read_args()
+    config = getConfig()
 
-    # TODO: works well only for small ledgers,
     ledger_data_dir = get_ledger_dir(args.node_name, args.client_name, args.network)
     read_copy_ledger_data_dir = None
     try:
-        read_copy_ledger_data_dir = make_copy_of_ledger(ledger_data_dir)
-        ledger = get_ledger(args.type, read_copy_ledger_data_dir)
+        # RocksDB supports real read-only mode and does not need to have a ledger copy.
+        if config.hashStore['type'].lower() != HS_ROCKSDB:
+            # NOTE: such approach works well only for small ledgers.
+            tmp = make_copy_of_ledger(ledger_data_dir)
```

Comment:
Maybe we should check, that shutil.copytree complete successfully? What if we have really big ledger, and will raised  "There is no space left on device" error?

Reply:
Hmm, I think it's already done. If exception is risen then we go directly to finally block. The read_copy_ledger_data_dir variable is empty in this case, so we do nothing, but a user will see the error message.